server/container/spring/src/main/resources/META-INF/org/apache/james/spring-server.xml

```diff
@@ -152,7 +152,7 @@
     <import resource="classpath:META-INF/spring/event-system.xml"/>
 
     <!-- Mailbox Copier -->
-    <bean id="mailboxcopier" class="org.apache.james.mailbox.tools.copier.MailboxCopierImpl"/>
+    <bean id="mailboxcopier" class="org.apache.james.mailbox.tools.quotas.MailboxCopierImpl"/>
```

Comment:
OUPS.

I believe that the copier have nothing to do in quotas package.

Reply:
Sorry not been careful with Intellij obviously...